components/LessonTitleCard.test.js

```diff
@@ -25,3 +51,23 @@ describe('LessonTitleCard component', () => {
     expect(container).toMatchSnapshot()
   })
 })
+
+describe('LessonTitleCard component on Review Page', () => {
+  const props = {
+    lessonCoverUrl: 'coverUrl',
+    lessonUrl: 'lessonUrl',
+    lessonTitle: 'Test Lesson',
+    lessonId: '0',
+    isPassed: true
+  }
+
+  test('Should render default layout for all screens', async () => {
+    global.window.innerWidth = 1080
+    let { container } = render(<LessonTitleCard {...props} />)
+    expect(container).toMatchSnapshot()
+
+    global.window.innerWidth = 500
+    ;({ container } = render(<LessonTitleCard {...props} />))
+    expect(container).toMatchSnapshot()
```

Comment:
Split this into two tests for wide and mobile layout. Tests should be testing a single thing only.

Reply:
Since markup does not change in this case and there is no specific functionality to test as it is static component, I've reduced the duplicate snapshot test and made it just check that the component gets rendered